src/modules/75-cf/pages/feature-flags/components/NoFeatureFlags.tsx

```diff
@@ -101,5 +101,11 @@ export const NoFeatureFlags: React.FC<NoFeatureFlagsProps> = ({
         width="570px"
       />
     </Container>
+  ) : (
+    <Container flex={{ justifyContent: 'center', alignItems: 'center' }} style={{ paddingTop: '150px' }}>
```

Comment:
Instead of using a `style` prop, you can just use the `padding` prop
```suggestion
    <Container flex={{ justifyContent: 'center', alignItems: 'center' }} padding={{ top: 150 }}>
```

Reply:
Lovely thanks :)

Reply:
for some reason the padding prop doesn't get picked up when I use it here? style seems to

Reply:
Maybe it only takes named sizes. Really `150` seems very arbitrary. What we really want is to position the component centred within the available vertical space. 150 might be ideal on your screen at the current viewport height, but it won't work for some others; potentially it will push it off the bottom of the page for some with smaller screens and  be way too close to the top for others. Does the parent stretch to the available height? In that case you should be able to set the `height` to 100%, if not don't worry about it and we can swing back around some time in the future (the vertical height on a lot of our pages are a bit funky).